crates/primitives/src/specification.rs

```diff
@@ -1,6 +1,9 @@
 #![allow(non_camel_case_types)]
```

Comment:
The way `SpecId`s are implemented changed, there's now a direct `enabled` fn on the `SpecId` enum itself. I've added support for this and two more tests mirroring the other ones on the `Spec` trait impl.